File: JAVA/Card.java
import java.util.HashMap;
import java.util.Map;
public class Card {
    private String type;
    private String status;
    private boolean present;
    private Map<String, String> info;

    public Card(String type, String status, Map<String, String> info) {
        this.type = type;
        this.status = status;
        this.present = !status.equals("lost");
        this.info = info;
    }

    public void displayInfo() {
        if (status.equals("lost")) {
            System.out.println("You have lost your " + type + ".");
        } else if (present) {
            System.out.println("===================================");
            System.out.println(type + ": Status - " + status);
            System.out.println("Card Information:");
            for (Map.Entry<String, String> entry : info.entrySet()) {
                System.out.println(entry.getKey() + ": " + entry.getValue());
            }
            System.out.println("===================================");
        } else {
            System.out.println("The " + type + " has been withdrawn. Please put it back.");
        }
    }

    public void withdraw() {
        if (present) {
            System.out.println("Withdrawing " + type + ".");
            present = false;
        } else {
            System.out.println(type + " already withdrawn.");
        }
    }

    public void putBack() {
        if (!present) {
            System.out.println("Putting back " + type + ".");
            present = true;
        } else {
            System.out.println(type + " already present.");
        }
    }
}

File: PYTHON/card.py
class Card:
    def __init__(self, card_type, status, info=None):
        self.type = card_type
        self.status = status
        self.present = status != "lost"
        self.info = info if info else {}

    def display_info(self):
        if self.status == "lost":
            print(f"You have lost your {self.type}.")
        elif self.present:
            print("===================================")
            print(f"{self.type}: Status - {self.status}")
            print("Card Information:")
            for key, value in self.info.items():
                print(f"{key}: {value}")
            print("===================================")
        else:
            print(f"The {self.type} has been withdrawn. Please put it back.")

    def withdraw(self):
        if self.present:
            print(f"Withdrawing {self.type}.")
            self.present = False
        else:
            print(f"{self.type} already withdrawn.")

    def put_back(self):
        if not self.present:
            print(f"Putting back {self.type}.")
            self.present = True
        else:
            print(f"{self.type} already present.")